### ISSUE TITLE
Can organization admins add new skills?

### ISSUE THREAD

**Issue by mjmilan** [HTBox/allReady#1719]
Can an organization admin add new skills to the system, and if not, should they be able to?

As new organizations come to use the system, they will want skills to be available that are relevant to the particular domains they work under. So, someone fitting a smoke alarm for example could, conceivably, have a skill of "Ladder Operation".

I am unable to confirm this directly as I cannot verify a new account to use as an organization admin.

If organization admins can add new skills directly, then I would suggest this issue be abandoned.

Tagging @ohmcgoo and @tonysurma for guidance here.

**Comment by tonysurma** [comment]
Org admins can create new skills at an org level to do as you describe, not sure off top of my head how much is complete or open issues but that is desired functionality and at least built into object model

**Comment by OhMcGoo** [comment]
I agree with Tony. This creates a potential problem of "category creep" meaning some Orgs may create skills so detailed that they may only apply to a single person (Smith County Left-Handed Power Drill Operators) but that's a user issue, not a coding issue.

**Comment by mjmilan** [comment]
I'm happy to close this to prevent anyone attempting code changes if you guys are...
